contracts/StorageManager.sol

```diff
@@ -117,6 +113,7 @@ contract StorageManager {
     }
```

Comment:
If I execute increaseAvailableCapacity before setOffer, I will create a offerRegistry with billingPeriods.length = 0
Is this correct?

Reply:
Yes, this is resolved now with `existingOffer` modifier. Thanks for the catch.